{"name": "test_new_success[test_info0]", "status": "broken", "statusDetails": {"message": "AttributeError: move_to requires a WebElement", "trace": "self = <test_cases.test_002_office.TestOffice object at 0x000002813043A388>\ntest_info = {'expected': '新增机构成功', 'name': '测试5527', 'officecode': '2317555', 'officename': '测试5527'}\nget_browser = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"4c088f8df47c21385dbadd0c7423632d\")>\n\n    @pytest.mark.parametrize('test_info',new_office_data_success)\n    @pytest.mark.flaky(reruns=1)\n    @allure.story(\"新增机构成功\")\n    def test_new_success(self,test_info,get_browser):\n        \"\"\"\n        新增机构成功\n        :param test_info:\n        :param get_browser:\n        :return:\n        \"\"\"\n        driver=get_browser\n        LoginPage(driver).login(User.username, User.pwd)\n        print(\"-------执行新增机构成功测试用例----------\")\n        expected = test_info['expected']\n>       OfficePage(driver).newOffice(test_info[\"name\"],test_info[\"officename\"], test_info[\"officecode\"])\n\ntest_002_office.py:31: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n..\\pages\\office_page.py:65: in newOffice\n    OfficePage.public(self)\n..\\pages\\office_page.py:58: in public\n    self.move_click(self.basic_management)\n..\\common\\base_page.py:162: in move_click\n    webdriver.ActionChains(self.driver).move_to_element(ele).click(ele).perform()\nC:\\Python37\\lib\\site-packages\\selenium\\webdriver\\common\\action_chains.py:273: in move_to_element\n    self.w3c_actions.pointer_action.move_to(to_element)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <selenium.webdriver.common.actions.pointer_actions.PointerActions object at 0x0000028130475F88>\nelement = None, x = None, y = None\n\n    def move_to(self, element, x=None, y=None):\n        if not isinstance(element, WebElement):\n>           raise AttributeError(\"move_to requires a WebElement\")\nE           AttributeError: move_to requires a WebElement\n\nC:\\Python37\\lib\\site-packages\\selenium\\webdriver\\common\\actions\\pointer_actions.py:42: AttributeError"}, "description": "\n        新增机构成功\n        :param test_info:\n        :param get_browser:\n        :return:\n        ", "attachments": [{"name": "log", "source": "0d1ebac0-7795-4085-b46f-233ab7a95269-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "test_info", "value": "{'name': '测试5527', 'officename': '测试5527', 'officecode': '2317555', 'expected': '新增机构成功'}"}], "start": 1600585939592, "stop": 1600585978738, "uuid": "16cb2272-e456-4c60-8add-fa063a10801c", "historyId": "80dadfe3e0b033006fd163a244e56761", "testCaseId": "96617d97cc50050747c41ecbb0d248e0", "fullName": "test_cases.test_002_office.TestOffice#test_new_success", "labels": [{"name": "story", "value": "新增机构成功"}, {"name": "parentSuite", "value": "test_cases"}, {"name": "suite", "value": "test_002_office"}, {"name": "subSuite", "value": "TestOffice"}, {"name": "host", "value": "LAPTOP-G986QGRS"}, {"name": "thread", "value": "5900-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_cases.test_002_office"}]}